_posts/2019-11-13-cockpit-207.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 207
+author: kkoukiou
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-207
+category: release
+summary: Cockpit with authentication and storage page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Cockpit's web server now accepts EC certificates
+
+To specify the [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) certificate
+the web service should use, simply drop a file with the extension .cert in the `/etc/cockpit/ws-certs.d` directory.
```

Comment:
That's a little ambiguous -- *all* certificates need to end in .cert. Maybe

The /etc/cockpit/ws-certs.d` directory can now contain [ECC] certificates in addition to the classic RSA ones.